packages/eslint-plugin-shopify/lib/config/rules/stylistic-issues.js

```diff
@@ -50,7 +50,7 @@ module.exports = {
   // Specify the maximum number of statements allowed per line
   'max-statements-per-line': ['warn', {max: 2}],
   // Require a capital letter for constructors
-  'new-cap': 'off',
+  'new-cap': ['error', {newIsCap: true, capIsNew: false}],
```

Comment:
Before now, if you were using the `es5` config, you wouldn't get any version of this rule at all (since the babel plugin is only brought in for `esnext`).